packages/zmarkdown/README.md

```diff
@@ -202,7 +202,7 @@ const opts = {
   extraRemarkPlugins: [
     {
       obj: remarkToc,
-      option: null /* remark plugin option. Null or omit to not configure it */,
+      option: undefined /* remark plugin option. undefined or omit to not configure it */,
```

Comment:
Thanks! Now the README needs to be updated accordingly, that's where I spotted this `null` idea